Add `readonly` examples and documentation (#189)
eclipsesource/jsonforms2-website: content/docs/uischema/controls.mdx

<ExampleWithRadioGroup />

### The `readonly` option

```json
options: {
  readonly: true
}
```

> When using the `readonly` option, you can disable the input to this field. See [example](/examples/readonly#ui-schema).

Reviewer: ```suggestion
When using the `readonly` option, you can disable the control or whole layout. See [example](/examples/readonly#ui-schema).
```
